src/ReactiveUI.Tests/CommandBindingTests.cs

```diff
@@ -170,15 +170,20 @@ public class CustomClickButton : Button
             this.CustomClick?.Invoke(this, EventArgs.Empty);
     }
 
-    public class CommandBindView : IViewFor<CommandBindViewModel>
+    public class CommandBindView : ReactiveObject, IViewFor<CommandBindViewModel>
```

Comment:
I think this should be an additional test VM (`ReactiveObjectCommandBindView`?). 

This is assuming it's a backward compatible change, e.g. ViewModels that do exist ahead of time on POCO views should still work.

Reply:
Done.